rss/plugin.go

```diff
@@ -10,6 +10,7 @@ import (
 func Plugin(ctx context.Context) *plugin.Plugin {
 	p := &plugin.Plugin{
 		Name:             "steampipe-plugin-rss",
+		DefaultShouldIgnoreError: isNotFoundError([]string{"403","404","500"}),
```

Comment:
@judell `404` is a typical not found response I'd expect, but can you please describe when you get 403 and 500? Do you have any query examples? Also, is this for RSS channels + items, or more useful with one table?

Reply:
@cody I was trying to iterate through a bunch of Mastodon feeds. You can form an RSS feed URL for any Mastodon user, e.g. https://social.coop/@judell, by appending .rss: https://social.coop/@judell.rss. Apparently some servers don't permit that, hence the 403s. And in at least one case I hit the cert error which was a 500.

Reply:
@judell Got it, so to confirm, you get the same errors with queries against the `rss_channel` and `rss_item` tables? Usually we put obvious not found errors, like `404` and `NotFound` in the default plugin `DefaultShouldIgnoreError`, and then add table specific ones as needed to avoid ignoring errors by mistake new tables where they don't have the special cases, e.g., if a new RSS table wants to actually throw 500 errors, we don't want to catch them in the plugin's `DefaultShouldIgnoreError`.

Also, for the cert error, if we have a known 500 error we want to ignore only in the case where it's related to the cert error, I think we should remove the `500` error code from a plugin or table `DefaultShouldIgnoreError` to avoid ignoring other `500` error codes.

Reply:
> @cody I was trying to iterate through a bunch of Mastodon feeds. You can form an RSS feed URL for any Mastodon user, e.g. https://social.coop/@judell, by appending .rss: https://social.coop/@judell.rss. Apparently some servers don't permit that, hence the 403s. And in at least one case I hit the cert error which was a 500.

Howdy! @‘d the wrong Cody on this one. Letting you know just in case it matters. Thanks!